mono/metadata/appdomain.c

```diff
@@ -44,7 +44,6 @@
 #include <mono/metadata/threads.h>
 #include <mono/metadata/threadpool.h>
 #include <mono/metadata/tabledefs.h>
-#include <mono/metadata/gc-internals.h>
 #include <mono/metadata/mono-gc.h>
```

Comment:
Some people have the opinion that files should include what they use and not depend on it being included from other include files.